diff --git a/dist/index.js b/dist/index.js
index 58aae7167a910753e9b59483cb79ae9634431b7d..f4281c0580ab5149ead171479346bae73c534160 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -83,6 +83,11 @@ var RedisStateAdapter = class {
       arguments: [lock.token]
     });
   }
+  async forceReleaseLock(threadId) {
+    this.ensureConnected();
+    const lockKey = this.key("lock", threadId);
+    await this.client.del(lockKey);
+  }
   async extendLock(lock, ttlMs) {
     this.ensureConnected();
     const lockKey = this.key("lock", lock.threadId);
